src/models/CommonModel.ts

```diff
@@ -164,7 +164,11 @@ export class CommonModel extends CommonSchema<CommonModel> {
         mergeTo.properties = mergeFromProperties;
       } else {
         for (const [propName, prop] of Object.entries(mergeFromProperties)) {
-          mergeTo.addProperty(propName, prop, originalSchema); 
+          if (mergeToProperties[`${propName}`] !== undefined) {
```

Comment:
I would consider using this if in `mergeCommonModels` method.

Reply:
I do not understand what you mean here, can you clarify it a bit more @czlowiek488?